src/PagesBody/ContestsPage/ContestsPage.js

```diff
@@ -0,0 +1,14 @@
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import ContestCard from '../../Molecules/ContestCard/ContestCard';
+import RANDOM_CONTESTS from '../../Data/Dummy-data/test-data-random-contests';
+
+const ContestsPage = () => {
+  return (
+    <ColumnWrapper paddingLeftRight={1} paddingTop={0.5}>
+      <ContestCard contestId={RANDOM_CONTESTS[0].id} contestIndex={0} />
+      <ContestCard contestId={RANDOM_CONTESTS[1].id} contestIndex={1} />
```

Comment:
What is the index for? Also - this shuold render card by mapping the contests array tht you should fetch on this page.